storage/scylla_api.c

```diff
@@ -0,0 +1,635 @@
+/*
+ * Copyright (C) 2019 BiiLabs Co., Ltd. and Contributors
+ * All Rights Reserved.
+ * This is free software; you can redistribute it and/or modify it under the
+ * terms of the MIT license. A copy of the license can be found in the file
+ * "LICENSE" at the root of this distribution.
+ */
+#include "scylla_api.h"
+#include <assert.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#define SCYLLA_API_LOGGER "SCYLLA_API"
+
+static logger_id_t scylla_api_logger_id;
+
+#define SCYLLA_LOG_ERROR(fmt, ...) \
```

Comment:
Please talk to @HowJMay for facilitate such techniques to extend the usage of loggers.

Reply:
Since new logger is merged into `develop` branch, please rework.